src/chains/ethereum/ethereum/src/state-manager.ts

```diff
@@ -1,20 +1,8 @@
 import SM from "@ethereumjs/vm/dist/state/stateManager";
+import { SecureTrie as Trie } from "merkle-patricia-tree";
 
-export class StateManager extends SM {
-  /**
-   * Removes accounts form the state trie that have been touched,
-   * as defined in EIP-161 (https://eips.ethereum.org/EIPS/eip-161).
-   */
-  // async cleanupTouchedAccounts() {
-  //   if (this._common.gteHardfork("spuriousDragon")) {
-  //     const touchedArray = Array.from(this._touched);
-  //     for (const address of touchedArray) {
-  //       const empty = await this.accountIsEmpty(address);
-  //       if (empty) {
-  //         this._cache.del({ buf: address } as any);
-  //       }
-  //     }
-  //   }
-  //   this._touched.clear();
-  // }
+export class GanacheStateManager extends SM {
```

Comment:
This whole class wasn't being used. So I commandeered it for my own nefarious purposes...

...which is just to get at the base class's private method `_getStorageTrie` without typescript complaining about it.

Reply:
I don't think we should update precompiles.ts, since it is using those just for the types.

_Theoretically_ this could be useful if we obtained a `stateManager` instance from something like `vm.stateManager`, and that instance's type returned only `StateManager` (despite it being a full `GanacheStateManager` with whatever _extra_ properties that may include). Since precompiles doesn't _need_ the extras from `GanacheStateManager`, using the narrower type is usually preferred.